lib/events.ts

```diff
@@ -34,28 +32,28 @@ const getMixpanel = _.once<any>(() => {
 	});
 });
 
-export function trackCommand(capitanoCli: Capitano.Cli) {
+export function trackCommand(commandSignature: string) {
```

Comment:
Previously, this function was Capitano-centric. This change makes it easier to reuse with either oclif or Capitano.